projects/sonar/src/app/deposit/editor/editor.component.ts

```diff
@@ -180,24 +180,24 @@ export class EditorComponent implements OnInit {
     if (this.deposit.metadata.publication.volume) {
       journal.push(
         this._translateService.translate('vol.') +
```

Comment:
Suggestion: `[this._translateService.translate('vol.'), this.deposit.metadata.publication.volume].join(' ')`